scripts/dasharo-deploy

```diff
@@ -736,7 +741,7 @@ restore() {
     # HCL report should be named as in dasharo-hcl-report so we can find
     # the package based on uuid saved in name, we need to check two options
     # with and without MAC address
-    uuid_string=`cat /sys/class/net/$(ip route show default | awk '/default/ {print $5}')/address`
+    uuid_string="cat /sys/class/net/$(ip route show default | awk '/default/ {print $5}')/address"
```

Comment:
@DaniilKl Backtis should be changed to `"$()"`, not to simple quotes. Otherwise, the command will not be executed, and we write the command to the `uuid_string`, not the result of it's execution.

Reply:
<Amended here e6d7198e4b16a01f5fd2b9fe62247eb84576d60a.